source/isaaclab/isaaclab/sim/simulation_cfg.py

```diff
@@ -92,8 +92,14 @@ class PhysxCfg:
     """Enable a second broad-phase pass that makes it possible to prevent objects from tunneling through each other.
     Default is False."""
 
-    enable_stabilization: bool = True
-    """Enable/disable additional stabilization pass in solver. Default is True."""
+    enable_stabilization: bool = False
+    """Enable/disable additional stabilization pass in solver. Default is False.
+
+    .. note::
+
+        Setting this flag to True should only be done if the simulation steps are very large (less than 30Hz).
+        Enabling this flag can lead to incorrect forces being read from the contact sensor.
```

Comment:
This should be a warning. Also a bit confusing: are forces just computed incorrectly or they are reported wrong? 

```suggestion

    .. warn::
    
        Enabling this flag may lead to incorrect contact forces report from the contact sensor.
```

Reply:
I have all the assets test passing locally now @kellyguo11